docs/4.0. Behaviour.md

```diff
@@ -248,11 +248,9 @@ A client cancels an activation by submitting a DELETE request to the resource `/
 
 ### Timestamps
 
-All timestamps in this API MUST be string formatted timestamps (\<seconds\>:\<nanoseconds\>) indicating time (absolute or relative) for activation. For relative times, the time '0:0' MUST be considered to be the time the request is received by the API.
+Timestamps indicate the absolute or relative time for activation and MUST be string formatted as (\<seconds\>:\<nanoseconds\>). Absolute timestamps MUST be based on TAI (which does not apply leap seconds) and must use SMPTE epoch (meaning 1970-01-01 00:00:00 TAI would be represented as '0:0').
```

Comment:
```suggestion
Timestamps indicate the absolute or relative time for activation and MUST be string formatted as \<seconds\>:\<nanoseconds\>. Absolute timestamps MUST be based on TAI (which does not apply leap seconds) and must use SMPTE epoch (meaning 1970-01-01 00:00:00 TAI would be represented as '0:0').
```

Reply:
52c86ad73d670874a4ee94140d7d0655258964c7 formats that as code